userguide/content/en/docs/Adding content/navigation.md

```diff
@@ -94,6 +94,16 @@ You can find a complete list of icons to use in the [FontAwesome documentation](
 
 Out of the box, if you want to use icons, you should define icons for all items on the same menu level in order to ensure an appropriate look. If the icons are used in a different way, individual CSS adjustments are likely necessary.
 
+### Add manual links to section menu
+
+By default the section menu is entirely generated from your section's pages. If you want to add a manual link to this menu, such as a link to an external site or a page in a different section of your site, you can do this by creating a placeholder page file in the doc hierarchy with the appropriate weight and some special parameters in its metadata (frontmatter). If a page has defined the parameter `manualLink` in its metadata, the generated link in the section menu for this page will be replaced by the value of `manualLink`. You can optionally also set a link title and target with the parameters `manualLinkTitle` and `manualLinkTarget` - for example if you want an external link to open in a new tab you can set the link target to `_blank`. 
+
+ You can use `manualLink` also to add a additional cross reference to another existing page of your site. But especially for intern links you can also use the parameter `manualLinkRelref` instead of `manualLink` to use the build in Hugo function [relref](https://gohugo.io/functions/relref/ "External link to official Hugo Docs"). If `relref` can't find a unique page in your site, Hugo will throw a error message.
+
+ {{% alert title="Attention" color="warning" %}}
+ Although the link in the menu is set according to the parameters `manualLink` or `manualLinkRelref`, the corresponding HTML page is still generated by Hugo. It is therefore advisable to specify the URL for the external link in the normal content and / or page description.
```

Comment:
Can you elaborate on this a bit  - I can see that the placeholder page will be generated, but I'm not sure what you mean by specifying the URL in the normal content?

Reply:
My thought was, if somebody opens the URL of such a placeholder page (e.g.. http://docsy-dev-manual-link.narrenfrei.at/docs/external-links/internal-link/), he don't know where the menu item originally links to. So a little hint on the "normal page content" (which is shown, when the placeholder page get's opened) would be helpful.

But I have no problem to delete these paragraph, if you think it's not so important.

Reply:
No, that makes sense - let me see if I can rephrase it a bit.